### PR TITLE
Change from Packet to Equinix Metal

### PR DESCRIPTION
**Pull request by dustinmiller**
Not sure if I missed anything, especially in netboot.xyz-metal.j2

### CODE REVIEW

**Comment by nshalman** [comment]
Perhaps add a line to `CHANGELOG.md`?

**Comment by dustinmiller** [comment]
> Perhaps add a line to `CHANGELOG.md`?
Definitely, added for us, thanks :smiley:

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [2.0.56]
```
Comment (nshalman) [inline CHANGELOG.md:7]: Might want to leave it in the Unreleased block and let the maintainers update a release version separately...

Comment (antonym) [reply CHANGELOG.md:7]: No worries, I'll fix before release.

**Review by antonym — APPROVED** [review]
Thanks, was going to switch this previously but figured it may break things on your side. I'll slip this in to the release this weekend.

**Comment by mmlb** [comment]
Actually I am a little worried about this. We set that for everything not just for netboot.xyz so if we do go with tinkerbell/boots#236 we can potentially be breaking other users. I'd rather have metal be an alias for packet, but then again idk if we'd ever actually change it so what would be the point...?

**Pull request merged**